src/uct/tcp/sockcm/sockcm_cm.h

```diff
@@ -0,0 +1,25 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2019.  ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#include <uct/base/uct_cm.h>
+
+#define UCT_SOCKCM_CM_PRIV_DATA_LEN  1024  /** Closest power of two to common MTU value in TCP (1500) */
```

Comment:
1. UCT_SOCKCM_CM_MAX_PRIV_LEN
2. maybe make it configurable?
3. not sure it's important why it would be power of 2?

Reply:
if it's configurable, what should be the default?
i thought maybe for alignments...

Reply:
1024 is ok, i would just remove the comment